tools/dds/dds-server/lrs-device-manager.h

```diff
@@ -0,0 +1,28 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <librealsense2/rs.hpp>  // Include RealSense Cross Platform API
+#include <unordered_map>
+
+namespace tools {
+
+// This class is in charge of handling a RS device: streaming, control..
+class lrs_device_manager
```

Comment:
Not sure about the name... let's discuss

Reply:
Done